proposals/devcontainer-collections.md

```diff
@@ -0,0 +1,110 @@
+# Proposal
+
+The proposal is to arrive to an schema where different communities and organizations can create "collections" of features/definitions that they manage, support and present to users/
+
+This collection data should have enough information so that different clients can search them and present them to users.
+
+For this we propose the existence of two different files:
+- Collection file: (devcontainer-collection.json) Groups of features/definitions that will be handled as the same origin (repository/publisher). Contains the metadata pertinent for each one, checksum and other information.
```

Comment:
For my own understanding, "microsoft/vscode-dev-containers" would be a "collection" in this context, correct?

Reply:
That's my understanding too.  This is a barebones "example" (which is what we'll turn the successors of vscode-dev-container into): https://github.com/joshspicer/devcontainer

Reply:
Renamed my example to `devcontainer-collection.json` to be consistent!

Reply:
Yeah, that is the idea.